chef_master/source/deprecations_json_auto_inflate.rst

```diff
@@ -0,0 +1,35 @@
+=====================================================
+Deprecation Warning: JSON Auto Inflate (CHEF-1)
+=====================================================
+`[edit on GitHub] <https://github.com/chef/chef-web-docs/blob/master/chef_master/source/deprecations_json_auto_inflate.rst>`__
+
+.. tag deprecations_json_auto_inflate
+
+Internally, the ``Chef::REST`` class attempts to guess which Chef class a JSON document relates too, and then automatically turns the JSON in to that class.
+
+.. end_tag
+
+This deprecation was added in Chef 12.7.2, and will be removed permanently in Chef 13.
```

Comment:
This should be clarified. We deprecated this *functionality* and the *functionality* will be removed in Chef 13.